autode/bracket/imagepair.py

```diff
@@ -454,16 +454,25 @@ def has_jumped_over_barrier(self) -> bool:
         Y. Liu, H. Qui, M. Lei, J. Chem. Theory. Comput., 2023
         https://doi.org/10.1021/acs.jctc.3c00151
         """
-        assert self.left_coord.g is not None and self.right_coord.g is not None
+        assert (
+            self.left_coords.g is not None and self.right_coords.g is not None
+        )
+
+        def cos_angle(vec1, vec2) -> float:
+            """Returns the cos(theta) between two vectors (1D arrays)"""
+            dot = float(np.dot(vec1, vec2))
+            dot /= np.linalg.norm(vec1)
+            dot /= np.linalg.norm(vec2)
```

Comment:
this is more readable for me but feel free to ignore if it isn't for you
```suggestion
            dot = (float(np.dot(vec1, vec2)) /
                       (np.linalg.norm(vec1) * np.linalg.norm(vec2)))
```

Reply:
@t-young31 Ok, I changed it slightly, does it read fine now?